### PR TITLE
Adapt test plan template

### PR DESCRIPTION
**Pull request by delvh**

### CODE REVIEW

File: dev-manuals/test-plans/template.md [hunk 1]
```diff
@@ -1,32 +1,31 @@
-# Name (`Test Case ID`)
+# $CATEGORY: $ACTION (`$TESTCASE_ID im Format "u.$(kebab-case $CATEGORY)-$NUMBER (from 1)"`)
```
Comment (maexled) [inline dev-manuals/test-plans/template.md:1]:
```suggestion
# $CATEGORY: $ACTION (`$TESTCASE_ID in format "u.$(kebab-case $CATEGORY)-$NUMBER (from 1)"`)
```
denglish

File: dev-manuals/test-plans/template.md [hunk 2]
```diff
@@ -1,32 +1,31 @@
...
 
-Test Case name pattern: u.XY(service identifier)-Z (sequential number, start at 1).  
-Version: vx.x, date of creation, description of change  
-Author: Author of the use case description  
-Tester: Use case tester
+Version: `v$X.$Y with $X >= 1, $Y >= 0`, `$NOW in YYYY-mm-dd`, `$CHANGE_DESCRIPTION like a git commit` \
+Author: Authors of this file, comma-separated
+(Delete this line in test cases!) Filename: `$(kebab-case u-$CATEGORY-$NUMBER-$ACTION).md`
```
Comment (maexled) [inline dev-manuals/test-plans/template.md:5]: To what is `(Delete this line in test cases!)` referenced to? Shouldn't be there line breaks?

File: dev-manuals/test-plans/template.md [hunk 3]
```diff
@@ -1,32 +1,31 @@
...
+2. Do `y`
 
 ## Alternative procedures
 
-Alternative procedures existing to the typical procedure
-Description by numbering of the executed
-alternative steps
+Do not show errors here, only intended alternatives in format
+1.1. `condition` - do `x2` \
+2.1. `condition 2` - do `y2`
```
Comment (maexled) [inline dev-manuals/test-plans/template.md:28]:
```suggestion
*Do not show errors here, only intended alternatives in format* \
1.1. `condition` - do `x2` \
2.1. `condition 2` - do `y2`
```
make the comment recursive or something that it shows it is a comment and not part of the procedure

File: dev-manuals/test-plans/template.md (hunk 2 above)
Comment (delvh) [reply dev-manuals/test-plans/template.md:5]: The following line with the filename.
That line is simply to clairfy how to write the filename.

File: dev-manuals/test-plans/template.md (hunk 3 above)
Comment (delvh) [reply dev-manuals/test-plans/template.md:28]:
```suggestion
<!-- Do not show errors here, only intended alternatives in format -->
1.1. `condition` - do `x2` \
2.1. `condition 2` - do `y2`
```

**Review by maexled — APPROVED** [review]

**Review by Gr33ndev — APPROVED** [review]

**Pull request merged**